### ISSUE TITLE
Documentation: Toggle on Message Content Intent

### ISSUE THREAD

**Issue by kjhosein** [thomasloupe/Slackord#17]
I just tried running Slackord today and although it would connect to the Discord server, running `!slackord` simply was't importing any messages. No error messages were thrown anywhere.

On a hunch based on this [notice from Discord about Message Intents](https://support-dev.discord.com/hc/en-us/articles/4404772028055-Message-Content-Privileged-Intent-FAQ), I toggled on _Message Content Intent_ in my bot (see screenshot).

![Screenshot 2022-09-01 3 40 11 PM](https://user-images.githubusercontent.com/3866191/187998989-d93d06f2-8b05-4387-830a-d6a31ee5b2d6.png)

I don't know if this is just a documentation addition, or whether a code update might be possible/necessary.

HTH!

**Comment by thomasloupe** [comment]
Hi @kjhosein,

Message intents are not required for non-verified bots. Are you saying that toggling this on fixed the issue? Additionally, since the bot is run per-user, and isn't a public bot, message intents are not required for Slackord.

The output of the bot itself should log to the console window. If the bot shows it's connected, then the next steps would be to look at the JSON file being parsed. If you'd like, you can DM me on Twitter with a link to the JSON and I can try on my own private development server to see if I can sort the issue out.

**Comment by SpiritBlossomAhri** [comment]
Hi. I'm in same situation, i got a message when i connect bot. 
<img width="637" alt="image" src="https://user-images.githubusercontent.com/74236661/188001620-580bca67-2725-4101-a297-09e8ff562137.png">
what is this mean?

**Comment by thomasloupe** [comment]
This just means I've set the bot to listen to all unprivileged events even though I am not actually asking for any data regarding them.

Most bots do this and it is not an error or a problem, just a notice that I've set the bot to listen to specific events, but there's no actual code listening to the event (because we really don't need it). It is easier to write `AllUnprivileged` than it is to specify each intent specifically, and `AllUnprivileged` means all intents that don't require additional permissions through the bot developer dashboard.

I will probably remove those intents to clear up the log window to prevent confusion.

**Comment by thomasloupe** [comment]
@kjhosein It is unclear to me right now whether or not the requirement is to enable message intents.
From what it appears, you should still enable message intents, and situations where previous bots had message intents automatically enabled, this will no longer be the case for new bot creators. I will update the README to reflect the policy changes Discord has put forth once I've had a moment to clarify the impact on Slackord.

**Comment by kjhosein** [comment]
Hi @thomasloupe - I should have said upfront that I don't fully understand the various Discord app & dev terminology and concepts. I just happened to solve the problem through a little bit of trial-and-error!

To recap:
* Yes, toggling the Message Intent to On solved the issue.
* I didn't get any console logging whatsoever.
* It didn't matter what JSON Slack export I threw at it - none worked until I turned on that setting.
* Since turning that setting on, I've been importing messages without this problem.
* When I previously created Discord apps+bots (prior to today, 1 Sep 2022), that Message Intent setting was _not_ toggled on by default.

I just wanted to let you know in case this Discord change affects Slackord in some material way.
I'll leave this issue open in case you wanted to add any more to it.

**Comment by thomasloupe** [comment]
Thanks, @kjhosein!

The README has been updated to include this information, as well as various changes to verbiage to be less complex, and more up-to-date.

In addition, Gateway Intents have been reworked so that the bot only uses intents it requires to post messages and read channels for the command trigger. This will get rid of the warnings about intents the bot isn't using but is subscribed to.

This, along with many bugfixes, code improvements, and user-error handling are now being built for 2.2.1.
See: https://github.com/thomasloupe/Slackord2/commit/ca481eb50b4897024e4864afbdd50bf26ad89158

In regards to the README update you provided, I have already updated the README to reflect these changes and will prefer to stick with those, though I appreciate your contribution and help in finding issues with Slackord2. Thank you! I'll go ahead and mark this as closed and release the new builds in the next few minutes.
